salt/metalk8s/kubernetes/cni/calico/deployed.sls

```diff
@@ -4321,75 +4323,72 @@ spec:
         # This container performs upgrade from host-local IPAM to calico-ipam.
         # It can be deleted if this is a fresh installation, or if you have already
         # upgraded to use calico-ipam.
```

Comment:
This comment seems to indicate that's only needed in a context where we had host-local IPAM configured, but that's never been the case for us, right? Can't we just remove it?

Reply:
Today, yes we could but maybe in the future this `calico-ipam -upgrade` command will do something "useful", I'm not sure it's worth removing it, it just add some "manual patch" to do on the calico upstream manifest